# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/StaticPrefs.h
# Commit: 1d9384906287
# Full Hash: 1d93849062870a8d492eeef2150ecea174cb6223
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-26 21:12:53
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r f5ebdcca640d -r 1d9384906287 modules/libpref/StaticPrefs.h
--- a/modules/libpref/StaticPrefs.h	Sun May 26 14:29:56 2019 +0000
+++ b/modules/libpref/StaticPrefs.h	Sun May 26 14:30:14 2019 +0000
@@ -13,6 +13,7 @@
 #include "mozilla/TypeTraits.h"
 #include "MainThreadUtils.h"
 #include <atomic>
+#include <cmath>  // for M_PI
 
 namespace mozilla {
 